code/model/modelcollide.cpp

```diff
@@ -354,9 +354,9 @@ static void mc_check_sphereline_face( int nv, vec3d ** verts, vec3d * plane_pnt,
 // +offset             vertex data. Each vertex n is a point followed by norm_counts[n] normals.     
 void model_collide_defpoints(ubyte * p)
 {
-	int n;
-	int nverts = w(p+8);	
-	int offset = w(p+16);	
+	uint n;
+	uint nverts = uw(p+8);	
```

Comment:
uw uses the old C/C++ method of casting, could we benefit from a C++11 `cast<>` templates?

Reply:
Probably, but there are *a lot * of these kinds of casts throughout these areas, beyond what we've touched, and I feel it might be better suited to a separate PR.

Reply:
Resolving this to be done in another PR